### PR TITLE
Shell casting

### PR DESCRIPTION
**Pull request by Dzierzan**
This adds shell casting for Copter. Initially I also wanted it for scouts, but due to being so small, ejected shells look huge next to them. And if desired, shells can be added for arty units "^DropMagazineCasing". This is also based on Graion changes

shells

### CODE REVIEW

File: mods/hv/rules/aircraft.yaml [hunk 1]
```diff
@@ -135,6 +136,9 @@ COPTER:
 		Sequence: propeller
 	WithTurretAttackAnimation:
 		Sequence: shoot
+	GrantConditionOnAttack:
```
Comment (GraionDilach) [inline mods/hv/rules/aircraft.yaml:139]: You don't need this.

File: mods/hv/rules/vehicles.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# License: CC-BY-SA-4.0
+d# License: CC-BY-SA-4.0
```
Comment (GraionDilach) [inline mods/hv/rules/vehicles.yaml:1]: Typo.

**Comment by Mailaender** [comment]
Can you squash the last 4 commits please?

**Comment by Mailaender** [comment]
I think you pulled in things which made it worse. See https://support.gitkraken.com/working-with-commits/squash/ for help.

**Comment by Dzierzan** [comment]
I did use that page to understand squashing. What doesn't work? I just tested it and it works fine.

**Comment by Mailaender** [comment]
Keep only the first 4 changes and squash the other commits into the 4th.

**Comment by GraionDilach** [comment]
I've did https://github.com/GraionDilach/OpenHV/tree/ShellCasting2 as an attempt. Feel free to merge that one if this is unsalvageable.

**Pull request closed without merging**